addons/website_sale/models/product.py

```diff
@@ -124,7 +126,14 @@ class ProductTemplate(models.Model):
     public_categ_ids = fields.Many2many('product.public.category', string='Website Product Category',
                                         help="The product will be available in each mentioned e-commerce category. Go to"
                                         "Shop > Customize and enable 'E-commerce categories' to view all e-commerce categories.")
-    product_image_ids = fields.One2many('product.image', 'product_tmpl_id', string='Images')
+
+    show_main_template_image_on_shop_page = fields.Boolean("Main Image on Shop page", default=True,
```

Comment:
show_image_on_shop_page
show_image_on_product_page
?

If there is not image in the name, it can be interpreted as showing or not the product itself?

Reply:
:+1: